src/screens/login/login-page.tsx

```diff
@@ -16,65 +14,97 @@ import {NavigationComponentProps} from 'react-native-navigation';
 import {UserPage} from '../users/user-page';
 import {ButtonComponent} from '../../components/button.component';
 import {InputComponent} from '../../components/input.component';
+import {
+  H1,
+  Container,
+  LoginInputWrapper,
+  TextError,
+} from '../../styles/screens.styles';
+
+interface GraphQLError {
+  graphQLErrors: Array<{
+    message: string;
+  }>;
+}
+
+export function isGraphQLError(error: unknown): error is GraphQLError {
+  return Array.isArray((error as any)?.graphQLErrors);
+}
```

Comment:
```suggestion
export function isGraphQLError(error: any): error is GraphQLError {
  return Array.isArray(error.graphQLErrors);
}
```

Semânticamente até faria sentido usar unkown, mas como o typescript não nos deixa acessar um campo de um tipo `unkown`, talvez fique mais prático e com leitura mais facilitada mesmo usar any desde o início.